__tests__/shared/didCommWithEthrDidFlow.ts

```diff
@@ -134,6 +134,106 @@ export default (testContext: {
       expect([...(resolution?.didDocument?.service || [])]).toEqual([])
     })
 
+    it('should add dummy service 2 to identifier', async () => {
```

Comment:
@nklomp I'm not sure why these changes are showing up in your PR.
Is it possible to rebase on top of our latest `next` branch, or merge it into your branch?

Reply:
Hmzz, odd, If i remember correctly it was rebased on @next. Will merge next in to fix

Reply:
thank you!